docs/reference/glossary.rst

```diff
@@ -0,0 +1,102 @@
+.. _glossary:
+
+.. glossary::
+
+========
+Glossary
+========
+
+.. contents::
+  :local:
+
+Feature release status
+======================
+
+.. note:: All Flocker documentation will refer the Production ready product, unless tagged Alpha or Beta.
+
+.. _production-ready-definition:
+
+Production ready
+   This product or feature is suitable for production use; expect API stability between versions; robust security; product stability for all supported uses.
+
+.. _beta-definition:
+
+Beta
+   This product or feature is approaching readiness; expect minor API instability between versions; incomplete security; product stability for most common uses.
```

Comment:
:question: I'd prefer either full sentences not split up by semicolons or a bulleted list